R/get_region_codes.R

```diff
@@ -35,7 +35,7 @@ rename_region_column <- function(data, country) {
     level_2_region_name <- switch(tolower(country),
                                   "belgium" = "province",
                                   "brazil" = "city",
-                                  "france" = "departement",
+                                  "france" = "department",
```

Comment:
You seem to have corrected something that is not a typo. You use the name in the relevant language for all other countries and "department" is "département" in French.

Reply:
My apologies, I should have checked properly! Corrected.